js/templates/modals/moderatorDetails.html

```diff
@@ -27,14 +27,14 @@
           <div class="tx5b clrT2"><%= ob.polyT('moderatorDetails.location') %></div>
         </div>
       </div>
-      <div class="rowDivV clrBr TODO"></div>
+      <div class="rowDivV clrBrBk TODO"></div>
       <div class="flexExpand TODO">
         <div class="flexCol flexCent">
           <%= ob.parseEmojis('👍') %> XX<% // placeholder for reputation %>
           <div class="tx5b clrT2"><%= ob.polyT('moderatorDetails.recommendations') %></div>
         </div>
       </div>
-      <div class="rowDivV clrBr"></div>
+      <div class="rowDivV clrBrBk"></div>
```

Comment:
This is fixing a wrong color on the dividing lines I noticed while making other changes (the same lines are used in the reputation page).